### ISSUE TITLE
`source=maxar` added to ways even if Maxar imagery is *not* used

### ISSUE THREAD

**Issue by habi** [facebookincubator/RapiD#396]
Version 1 of [way/947830423](https://www.openstreetmap.org/way/947830423/history) was added with RapiD 1.1.4, with imagery *not* including Maxar [1].
While tagging the `source` on an object is not officially deprecated, I think that good practice is tagging it on the changeset.
(RapiD actually does that with `source=mapwithai` :confused:)

I confirmed myself that the (in my opinion wrong) source-tag would be added to a way (wrongly detected under a power line) somewhere in eastern Switzerland via [this RapiD link](https://mapwith.ai/rapid#background=swisstopo_swissimage&datasets=fbRoads,msBuildings&disable_features=boundaries&id=w207981553&map=19.46/46.73508/10.08251).

Can the `source=maxar`-Tag *not* be added to ways added with RapiD?

[1]: The changeset tags include `imagery_used=Bing aerial imagery;OpenStreetMap GPS traces;swisstopo SWISSIMAGE;Esri World Imagery;swisstopo SWISSALTI3D;Esri World Imagery (Clarity) Beta`

**Comment by habi** [comment]
Any input on this?

**Comment by bhousel** [comment]
Hey @habi  - sorry I missed this the first time around.

I sort of agree that adding `source=maxar` is a bit odd, but our team is using it for analytics to determine that the provenance of these ways came from the Meta Roads dataset, which was trained on the Maxar imagery at the time..  
(this `source` tag isn't related to the `imagery_used` changeset tag)

Since then, we have added a lot of datasets, so we've standardized on much better source tag values..
For example, we add `source=microsoft/BuildingFootprints` for features from Microsoft Building Footprints dataset,
or `source=esri/<dataset_id>` (e.g. `source=esri/Bucks_County_PA_Buildings`) for features from any of the Esri open datasets.  Going forward we are committed to using better values than `maxar`.

Also,
>While tagging the source on an object is not officially deprecated, I think that good practice is tagging it on the changeset.

We need to add the `source` tag on the feature instead of the changeset because we expect users to add a mix of features, some traced by hand and some suggested by rapid, so we want to track this usage separately.  It wouldn't be accurate or fair to assume that everything the user adds in their editing session came from a rapid dataset.
